docs/installation/ha-installation-with-cli.md

```diff
@@ -0,0 +1,69 @@
+---
+title: High Availability Installation Using CLI
+---
+
+This documentation serves as a comprehensive guide for deploying Karmada in a high
+availability (HA) configuration using the `karmadactl` CLI tool.
+
+## Prerequisites
+
+- A Kubernetes cluster with multiple worker nodes (it's recommended to have at least
+  three worker nodes).
+- A valid `kube-config` file to access your Kubernetes cluster.
+- The `karmadactl` command-line tool or `kubectl-karmada` plugin installed.
+
+  :::note
+
+  For installing `karmadactl` command line tool refer to the [Installation of CLI Tools](/docs/next/installation/install-cli-tools#one-click-installation) guide.
+
+  :::
+
+## Installation
+
+Follow these steps to install Karmada with high availability:
+
+1. Use the following command to install the Karmada in HA:
+
+  ```bash
+  sudo karmadactl init --karmada-apiserver-replicas 3 --etcd-replicas 3 --etcd-storage-mode PVC --storage-classes-name <storage-classes-name> --kubeconfig <path-to-kube-config>
+  ```
```

Comment:
```suggestion
   ```bash
   sudo karmadactl init --karmada-apiserver-replicas 3 --etcd-replicas 3 --etcd-storage-mode PVC --storage-classes-name <storage-classes-name> --kubeconfig <path-to-kube-config>
   ```
```